pkg/scheduler/framework/framework.go

```diff
@@ -640,21 +664,19 @@ func (f *framework) createBindings(ctx context.Context, toCreate []*placementv1b
 		// TO-DO (chenyu1): Add some jitters here to avoid swarming the API when there is a large number of
 		// bindings to create.
 		if err := f.client.Create(ctx, binding); err != nil {
-			return controller.NewAPIServerError(false, fmt.Errorf("failed to create binding %s: %w", binding.Name, err))
+			return controller.NewCreateIgnoreAlreadyExistError(fmt.Errorf("failed to create binding %s: %w", binding.Name, err))
```

Comment:
Hi Ryan! Just a nit: though it may not happen in most of the cases, I am afraid that AlreadyExistsError cannot be ignored for this one, as the name is generated randomly each time